server/src/test/java/io/druid/query/dimension/LookupDimensionSpecTest.java

```diff
@@ -70,12 +71,13 @@
   public void testSerDesr(DimensionSpec lookupDimSpec) throws IOException
   {
     ObjectMapper mapper = new DefaultObjectMapper();
+    mapper.registerSubtypes(new NamedType(LookupDimensionSpec.class, "lookup"));
     InjectableValues injectableValues = new InjectableValues.Std().addValue(
         LookupReferencesManager.class,
         LOOKUP_REF_MANAGER
     );
     String serLookup = mapper.writeValueAsString(lookupDimSpec);
-    Assert.assertEquals(lookupDimSpec, mapper.reader(LookupDimensionSpec.class).with(injectableValues).readValue(serLookup));
+    Assert.assertEquals(lookupDimSpec, mapper.reader(DimensionSpec.class).with(injectableValues).readValue(serLookup));
```

Comment:
part of PR?

Reply:
Needed due to the changed Jackson version.